geps/gep-2648/index.md

```diff
@@ -1,12 +1,17 @@
 # GEP-2648: Direct Policy Attachment
 
 * Issue: [#2648](https://github.com/kubernetes-sigs/gateway-api/issues/2648)
-* Status: Provisional
+* Status: Memorandum
```

Comment:
I think this should be changed to "Declined", as it's no longer valid. "Memorandum" implies that the document is still in effect, but this has been wholly superseded by updates to GEP-713 in this PR.

Reply:
This was marked resolved, but seems to have not been updated:

```suggestion
* Status: Declined
```